.github/workflows/ci.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         os: [macOS-latest, ubuntu-latest]
         openmm: ["true", "false"]
         python-version: ["3.10", "3.11", "3.12"]
-        pint-version: ["0.21", "0.22", "0.23"]
+        pint-version: ["0.24"]
```

Comment:
Unless there's a good reason to support older versions, I want to keep on the most recent minor version (though this is from several months ago)